### DISCUSSION TITLE
Organization stats ?

### DISCUSSION THREAD

**Discussion by DarkXero-dev** [anuraghazra/github-readme-stats#2312] [Q&A] [answered]
Is there a way to only include Organization stats ? username=organizationname does not work.. 

Thanks

**Comment by rickstaa** [comment, marked as answer]
@TechXero Thanks for your question. This is one of our most requested features but has not yet been released (see https://github.com/anuraghazra/github-readme-stats/issues/1935). You can, however, already use it (see https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-1321896111).

**Reply by DarkXero-dev** [reply]
Is that correct ? If so why is it showing my Username not Org. name ? 
```
https://github-readme-stats-git-masterorgs-github-readme-stats-team.vercel.app/api?username=techxero&include_orgs=true&show_icons=true&theme=nightowl&locale=en
```
[Organization](https://github.com/xerolinux)

**Reply by rickstaa** [reply]
@TechXero , thanks for your question. The feature request I linked (i.e. #1) is about including stats from organization repositories for users that have contributed to these repositories. This feature has not yet been released but can be used as explained in https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-1329809068. Your stats will be included if you are a collaborator to such a repository when setting the `include_orgs` variable.

The feature you are asking for, which I presume is organisation/team stats, is tracked in #654. You can show your support there. If enough people show interest in such a card, it will come in our top 10 feature requests, and I will take a look (see https://github.com/anuraghazra/github-readme-stats/issues/1935).

**Reply by DarkXero-dev** [reply]
I am the owner of all orgs n main account but I understand. Using the `include_orgs` variable will combine everything from profile and orgs.. Or am I wrong ?

**Reply by rickstaa** [reply]
@TechXero, thanks for your question. The `include_orgs` tag currently works as follows. When enabled, it adds the `COLLABORATOR` `repositoryAffiliation` filter to the GraphQL query:

https://github.com/anuraghazra/github-readme-stats/blob/06aee4f12c1f8675d961405d78e841cad40a25ef/src/fetchers/stats-fetcher.js#L25

As explained in [GitHubs' GraphQL documentation](https://docs.github.com/en/graphql/reference/enums#repositoryaffiliation), this will include repositories of the requested user that are found in organizations. It, therefore, does only support user accounts and not organization accounts. The feature request to also show stats about organization accounts can be found at https://github.com/anuraghazra/github-readme-stats/issues/654. Hope that answers your question.
